ce/ce/installers/util.ts

```diff
@@ -22,13 +22,15 @@ export function artifactFileName(name: string, install: Installer, extension: st
 }
 
 export function applyAcquireOptions(options: AcquireOptions, install: Verifiable): AcquireOptions {
-  if (install.sha256) {
-    return { ...options, algorithm: 'sha256', value: install.sha256 };
+  let sha256 = install.sha256;
+  if (sha256 !== null && sha256 !== undefined) {
+    return { ...options, algorithm: 'sha256', value: sha256.toString() };
```

Comment:
we do know that `sha256` is a `string`, so we don't need to write `sha256.toString()`

Reply:
Unfortunately, we don't know that it is a string. This was in fact the bug, the original repro did:

```
"install": {
  "unzip": "some uri",
  "sha256": 0
}
```

and 0 got interpreted as 'falsy' and no hash check :(

Reply:
🌈 Javascript!

Reply:
Apparently this is the whole `any` coerces to any type thing, ugh.

Reply:
>Shouldn't the Verifiable type mandate that this is/is not a string though?

There doesn't appear to be a Typescript 'check that this thing matches that interface' feature.